attachment_edit/static/src/css/attachment_edit.css

```diff
@@ -0,0 +1,22 @@
+.openerp .oe_sidebar .oe_dropdown_menu li .oe-sidebar-attachment-edit
+{
+    position: absolute;
```

Comment:
Yes, the discussion was in https://github.com/OCA/knowledge/issues/87. OK, let's keep this and with the next added button, we make the move.